templates/certificate_autorenewal_configuration.yaml.erb

```diff
@@ -0,0 +1,23 @@
+# Managed by Puppet
+# puppet-sslcert
+---
+auto_cert_renewal_certificates:
+        <%= @name %>:
+                certificate_file: "<%= @certificate_file %>"<% unless @certificate_mode.empty? %>
```

Comment:
Is het de bedoeling dat hier "" staat als certificate_mode leeg is? is wat onduidelijk imho op deze manier. 
Misschien eerst in puur ruby je hash opbouwen en dan gewoon to_yaml gebruiken.

Reply:
2 problemen.
Hoe doe ik dat? (ik weet niet, hoe ik met mijn if structuren in hashes kan werken)
klein vb zou tof zijn.
Gaat dit werken in puppet3 code? Ik wil de code hergebruiken.